lib/classes/Variables.js

```diff
@@ -344,11 +344,24 @@ class Variables {
     }, valueToPopulate);
   }
 
-  warnIfNotFound(variableString, valueToPopulate) {
+  deleteNullEnvironmentVariables() {
```

Comment:
@HyperBrain thanks for getting back.

Yes, I agree. Putting it into the packaging phase sounds like the best idea! 👍